### ISSUE TITLE
restic and fluent-bit fail to start on controller nodes due to cni plugin not initialized

### ISSUE THREAD

**Issue by tyler-public** [darxkies/k8s-tew#29]
Commands run:
```
k8s-tew node-add -n node0-i 192.168.10.XX -x 0 -l controller
k8s-tew node-add -n node1 -i 192.168.10.XX -x 1 -l controller
k8s-tew node-add -n node2 -i 192.168.10.XX -x 2 -l controller
k8s-tew node-add -n node3 -i 192.168.10.XX -x 3 -l worker,storage
k8s-tew configure --public-network 192.168.10.0/24
k8s-tew generate --parallel --pull-images
k8s-tew deploy --import-images
```

Log output for `containerd.log`:
```
root@node0:/var/log/k8s-tew# cat containerd.log | grep error
time="2020-09-01T21:20:10.292933160Z" level=info msg="skip loading plugin \"io.containerd.snapshotter.v1.btrfs\"..." error="path /var/lib/k8s-tew/containerd/io.containerd.snapshotter.v1.btrfs (ext4) must be a btrfs filesystem to be used with the btrfs snapshotter: skip plugin" type=io.containerd.snapshotter.v1
time="2020-09-01T21:20:10.292979721Z" level=warning msg="failed to load plugin io.containerd.snapshotter.v1.devmapper" error="devmapper not configured"
time="2020-09-01T21:20:10.294183989Z" level=info msg="skip loading plugin \"io.containerd.snapshotter.v1.zfs\"..." error="path /var/lib/k8s-tew/containerd/io.containerd.snapshotter.v1.zfs must be a zfs filesystem to be used with the zfs snapshotter: skip plugin" type=io.containerd.snapshotter.v1
time="2020-09-01T21:20:10.294224005Z" level=warning msg="could not use snapshotter devmapper in metadata plugin" error="devmapper not configured"
time="2020-09-01T21:20:10.305771036Z" level=error msg="Failed to load cni during init, please check CRI plugin status before setting up network for pods" error="cni config load failed: no network config found in /etc/k8s-tew/cni: cni plugin not initialized: failed to load cni config"
```
Log output for `kubelet.log` is full of:
```
E0901 22:13:21.631931  185025 pod_workers.go:191] Error syncing pod 5c7fa076-8834-41c5-b7b8-90f146eb5bcc ("restic-dsq5t_backup(5c7fa076-8834-41c5-b7b8-90f146eb5bcc)"), skipping: network is not ready: runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: cni plugin not initialized
```

Using `k8s-tew` version `2.4.0-beta.5`, running Ubuntu 20.04.1 LTS, this is consistent across all 3 controller nodes deployed

**Comment by darxkies** [comment]
Can you post the list of pods? The calico-nodes were probably not started. How long did you wait?

**Comment by tyler-public** [comment]
This has resolved itself after fixing DNS in #31 

Closing
